lib/charms/mysql/v0/mysql.py

```diff
@@ -677,14 +677,15 @@ def is_instance_in_cluster(self, unit_label: str) -> bool:
             )
             return False
 
-    def get_cluster_status(self) -> dict:
+    def get_cluster_status(self) -> Optional[dict]:
```

Comment:
@swalladge this change would be better suited on the [mysql vm operator](https://github.com/canonical/mysql-operator), that's where this library file resides.

I'm replicating it, so don't worry doing so for this change.

Reply:
Great catch @paulomach